base/linalg/dense.jl

```diff
@@ -5,6 +5,7 @@
 ## BLAS cutoff threshold constants
 
 const SCAL_CUTOFF = 2048
+const TRIA_CUTOFF = 2048
```

Comment:
Maybe call it `ISONE_CUTOFF` to following the naming scheme of the other constants.